digital_land_frontend/jinja.py

```diff
@@ -49,35 +23,8 @@ def setup_jinja():
     env = jinja2.Environment(loader=multi_loader, autoescape=True)
 
     # register jinja filters
-    env.filters["group_id_to_name"] = group_id_to_name_filter
-    env.filters["organisation_id_to_name"] = organisation_id_to_name_filter
-    env.filters["organisation_id_to_url"] = organisation_id_to_url_filter
-    env.filters["geography_to_name"] = geography_to_name_filter
-    env.filters["geography_to_url"] = geography_to_url_filter
-    env.filters["geography_to_geometry_url"] = geography_to_geometry_url_filter
-    env.filters["clean_slug"] = strip_slug
-    env.filters["make_link"] = make_link
-    env.filters["is_list"] = is_list
-    env.filters["is_historical"] = is_historical
-    env.filters["contains_historical"] = contains_historical
-    env.filters["get_geometry_url"] = get_geometry_url_filter
-    env.filters["policy_to_development_plan"] = policy_to_development_plan_filter
-    env.filters["policy_mapper"] = policy_mapper.filter
-    env.filters["dev_doc_mapper"] = dev_doc_mapper.filter
-    env.filters["plan_type_mapper"] = plan_type_mapper.filter
-    env.filters["policy_category_mapper"] = policy_category_mapper.filter
-    env.filters["developer_agreement_mapper"] = developer_agreement_mapper.filter
-    env.filters[
-        "developer_agreement_contribution_mapper"
-    ] = developer_agreement_contribution_mapper.filter
-    env.filters[
-        "developer_agreement_type_mapper"
-    ] = developer_agreement_type_mapper.filter
-    env.filters["dl_category_mapper"] = category_mapper_router.route
-    env.filters["key_field"] = key_field_filter
-    env.filters["github_line_num"] = github_line_num_filter
-    env.filters["reference_mapper"] = reference_filter
-    env.filters["total_items"] = total_items_filter
+    register_basic_filters(env)
+    register_mapper_filters(env, view_model)
```

Comment:
`setup_jinja()` is used by all our render repos. Some repos for rendering the record pages, some for rendering other parts of the site that probably don't need the view_model to be passed in (e.g. the map or the project pages). 

So how about making the view model optional?

E.g.
```
def setup_jinja(view_model):
...
    if view_model is not None:
        register_mapper_filters(env, view_model)
```

Reply:
Yes good catch!